### PR TITLE
fix: Removed leftover scripts

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

Removed leftover scripts that are redundant and not referenced

### CODE REVIEW

File: avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1
Comment (eriqua) [inline avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1:1]: Hey there this is already part of #717

Comment (AlexanderSehr) [reply avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1:1]: I know - but should it @eriqua?

Comment (eriqua) [reply avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1:1]: Not a problem for me if @rahalan is fine with that

Comment (AlexanderSehr) [reply avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1:1]: Why would he not?

Comment (eriqua) [reply avm/utilities/e2e-template-assets/scripts/.scripts/Copy-VhdToStorageAccount.ps1:1]: I'm sure he will

**Review by ChrisSidebotham — APPROVED** [review]
Approved - @rahalan update Vm Branch with main changes

**Pull request merged**